src/annotator/test/test_files/test.vrt

```diff
@@ -0,0 +1,16 @@
+<s>
+This	DT	this	-	nsubj	PRON
```

Comment:
This is not a valid vrt, though, right? Due to the "-", where does it come from?

Reply:
If I'm not mistaken this is due to the fact that the corresponding Tag has not been defined. I think we need something as a placeholder though right? The "-" is defined in the base.out_object.grab_ functions. I can change it to a whitespace or anything else.

Reply:
If it is a whitespace it will be encoded as __UNDEF__ in CWB